src/lib/modules/ui/components/aeris-ui-input.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div aeris-ui-input>
+    <i v-if="icon" :class="icon" />
+    <input
+      :name="name"
+      :aria-label="ariaLabel"
+      :placeholder="placeholder"
+      type="text"
+      autofocus
+      @change.prevent="updateValue"
+      @keyup.enter="updateValue"
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  name: "aeris-ui-input",
+
+  props: {
+    icon: {
+      type: String,
+      default: ""
+    },
+    name: {
+      type: String,
+      default: ""
+    },
+    placeholder: {
+      type: String,
+      default: ""
+    },
+    ariaLabel: {
+      type: String,
+      default: ""
+    }
+  },
+
+  methods: {
+    updateValue: _.debounce(function(event) {
```

Comment:
ES6 : `updateValue: _.debounce(event => this.$emit("input", event.target.value), 100)`